server/src/main/java/org/opensearch/transport/TransportService.java

```diff
@@ -906,6 +923,54 @@ public String toString() {
         }
     }
 
+    private String createSpanName(String action, Transport.Connection connection) {
+        return action + " " + (connection.getNode() != null ? connection.getNode().getHostAddress() : null);
+    }
+
+    private <T extends TransportResponse> TransportResponseHandler<T> getTransportResponseHandler(
```

Comment:
Please move it to  `TraceableTransportResponseHandler` (as we did for  `TraceableActionListener` & co)

Reply:
Since, its being used at one place only so kept like that. Moving now.